poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.8.2 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.7.1 and should not be changed by hand.
```

Comment:
Why did you downgrade poetry?

Reply:
I used an old venv with poetry 1.7.1 but that has no impact